src/App.vue

```diff
@@ -633,7 +635,7 @@ export default class App extends Mixins(
     await this.checkAuth(this.getBusinessId).catch(error => {
       console.log('Auth error =', error) // eslint-disable-line no-console
       this.accountAuthorizationDialog = true
-      throw error // go to catch()
```

Comment:
This method doesn't know anything about a catch. (That's in the parent component.)